{"name": "test_mark_function_by_process_tag_working_exception", "status": "broken", "statusDetails": {"message": "Exception: Autogenerated Exception.", "trace": "self = <tests.test_report.TestClass object at 0x7f2e615790f0>\n\n    @pytest.mark.test_process_marks(\"Working\")\n    def test_mark_function_by_process_tag_working_exception(self):\n>       raise Exception(\"Autogenerated Exception.\")\nE       Exception: Autogenerated Exception.\n\ntests/test_report.py:20: Exception"}, "attachments": [{"name": "log", "source": "4c77ddbd-0b30-4487-8779-8b525b6a1d23-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "f50836dc-0685-4b1b-bb0c-a033169cf6b9-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "3740847a-1cd9-45dc-b834-6317b4f6e183-attachment.txt", "type": "text/plain"}], "start": 1554843314029, "stop": 1554843314030, "uuid": "dc4084a9-424f-4443-806d-9d3aa0c4e29c", "historyId": "30ba1d34e3af98868088428607c07b34", "fullName": "tests.test_report.TestClass#test_mark_function_by_process_tag_working_exception", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_report"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "travis-job-72aa73d9-316a-482c-a718-fe1ab133a4a8"}, {"name": "thread", "value": "3720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_report"}]}